esp8266/scripts/apa102.py

```diff
@@ -0,0 +1,24 @@
+# APA102 driver for MicroPython on ESP8266
+
+from esp import apa102_write
+
+class APA102:
+    def __init__(self, clock_pin, data_pin, n):
+        self.clock_pin = clock_pin
+        self.data_pin = data_pin
+        self.n = n
+        self.buf = bytearray(n * 4)
+
```

Comment:
Do you need to set them as output pins here?
`self.clock_pin.init(pin.OUT)`
`self.data_pin.init(pin.OUT)`